docs/user-guide/workflow-setup.md

```diff
@@ -168,7 +168,18 @@ Will prevent any content from the domains listed in [Steven Black's Unified Host
 
 ### User Agent
 
-Sets the browser's user agent in outgoing requests to the specified value. If left blank, the crawler will use the browser's default user agent.
+Sets the browser's [user agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) in outgoing requests to the specified value. If left blank, the crawler will use the Brave browser's default user agent. For a list of common user agents see [useragents.me](https://www.useragents.me/).
+
+??? example "Using custom user agents to get around restrictions"
+    Despite being against best practices, some websites will block specific browsers based on their user agent: a string of text that browsers send web servers to identify what type of browser or operating system is requesting content. If Brave is blocked, using a user agent string of a different browser (such as Chrome or Firefox) may be sufficient to convince the website that a different browser is being used.
+
+    User agents can also be used to voluntarily identify your crawling activity, which can be useful when working with a website's owners to ensure crawls can be completed successfully. We recommend using a user agent string similar to the following, replacing the `orgname` and URL comment with your own:
+
+    ```
+    Mozilla/5.0 (Windows NT 10.0; Win64; x64) orgname.browsertrix (+https://example.com/crawling-explination-page)
```

Comment:
I'd recommend that we use a user agent that is more reflective of what browsers actually send here. Maybe it would be sufficient to tell people to pick a user agent they want to use from usersagents.me and just add the ` orgname.browsertrix (+https://example.com/crawling-explination-page)` bit to the end of it?

Reply:
guh after sending you that chat I didn't update it lol